src/app/services/views/ServiceListView.vue

```diff
@@ -57,7 +57,7 @@
               </template>
 
               <template #serviceType="{ rowValue }">
-                {{ rowValue || t('common.collection.none') }}
+                {{ rowValue || 'internal' }}
```

Comment:
Should this use `t`, seeing as it was using `t` previously? Isn't this a `http.api.value`?

Reply:
This should definitely not use t. This is a value that may never be different.

Reply:
Yeah gotcha, but its a value that is like this because it comes from the `http.api` right? All the others like this use `t` (well all the others up until now)

Reply:
This is an API value that should be only every represented as the literal string `'internal'`. In other cases, changing the value for aesthetic reasons might be fine. Here, I really think this *must* not happen. This should only ever be allowed to be exactly `'internal'`.